sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/datalake_options.hpp

```diff
@@ -21,16 +21,44 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
     using PathHttpHeaders = Blobs::Models::BlobHttpHeaders;
     using ListFileSystemsIncludeFlags = Blobs::Models::ListBlobContainersIncludeFlags;
     using SignedIdentifier = Blobs::Models::SignedIdentifier;
+    using EncryptionAlgorithmType = Blobs::Models::EncryptionAlgorithmType;
   } // namespace Models
 
   using DownloadFileToOptions = Blobs::DownloadBlobToOptions;
   using GetUserDelegationKeyOptions = Blobs::GetUserDelegationKeyOptions;
 
+  /**
+   * @brief Wrapper for an encryption key to be used with client provided key server-side
+   * encryption.
+   */
+  struct EncryptionKey final
+  {
+    /**
+     * @brief Base64 encoded string of the AES256 encryption key.
+     */
+    std::string Key;
+
+    /**
+     * @brief SHA256 hash of the AES256 encryption key.
+     */
+    std::vector<uint8_t> KeyHash;
+
+    /**
+     * @brief The algorithm for Azure Blob Storage to encrypt with.
+     */
+    Models::EncryptionAlgorithmType Algorithm;
+  };
+
```

Comment:
Why not just typedef the one in blob sdk?

Reply:
That's a good idea. I will change it.